Chrome_Extenstion/FrontEndFiles/HTML/newtab.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>New Tab</title>
+    <link rel="stylesheet" href="../CSS/newtab.css">
+</head>
+<body id="main_body" >
```

Comment:
I was tought to always try to leavy the Body and head tags alone and works with internal divs. for example and as  the way i did it was to do a 

<body>
<main id = "blah blah blah.">
work inside here. 
<main/>
<body/>

I guess that as body is a standard structure in the HTML file we where thought like this.

Reply:
`<body>
<main>
work inside here
<main/>
<body/>`

Reply:
not being able to put code in the text. but i guess you get the idea.